tests/unit/profiles-management/helpers/test_k8s.py

```diff
@@ -0,0 +1,31 @@
+from lightkube.generic_resource import GenericNamespacedResource
+from lightkube.models.meta_v1 import ObjectMeta
+
+from profiles_management.helpers.k8s import get_annotations, to_rfc1123_compliant
+
+
+def test_annotations():
+    resource = GenericNamespacedResource(
+        metadata=ObjectMeta(name="test", annotations={"test": "value"})
+    )
+
+    assert get_annotations(resource) == {"test": "value"}
+
+
+def test_no_annotations():
+    resource = GenericNamespacedResource(metadata=ObjectMeta(name="test"))
+
+    assert get_annotations(resource) == {}
```

Comment:
Similar to my previous comment, you should use a parametrised test.

Reply:
Sure!
e51bbfd